shell/platform/android/test/io/flutter/Utils.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter;
+
+import static org.junit.Assert.assertTrue;
+
+import android.content.res.Configuration;
+import android.os.Build;
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.Locale;
+
+public class Utils {
```

Comment:
Call this TestUtils so it will not collide with any Utils class in the io.flutter embedding packages

Reply:
Makes sense. Done